pyoverkiz/client.py

```diff
@@ -69,15 +69,6 @@
 from pyoverkiz.obfuscate import obfuscate_sensitive_data
 from pyoverkiz.types import JSON
 
-
-async def relogin(invocation: dict[str, Any]) -> None:
-    await invocation["args"][0].login()
```

Comment:
Change the type for `invocation` to `Mapping[str, Any]`. Same for `refresh_listener`.
`backoff` changed it's typing to except a `TypedDict` as first argument. `dict[str, Any]` would be incompatible.

Reply:
Thanks @cdce8p, I missed your comment before!

Reply:
```pyoverkiz/client.py:719: error: Argument "on_backoff" to "on_exception" has incompatible type "Callable[[Mapping[str, Any]], Coroutine[Any, Any, None]]"; expected "Union[Callable[[Details], None], Iterable[Callable[[Details], None]]]"  [arg-type]```

This still has an issue :-)